RxSwift/Subjects/Variable.swift

```diff
@@ -10,7 +10,7 @@
 ///
 /// Unlike `BehaviorSubject` it can't terminate with error, and when variable is deallocated
 /// it will complete it's observable sequence (`asObservable`).
-public final class Variable<Element> {
+public final class Variable<Element>: ObservableConvertibleType {
```

Comment:
I got your point, but I just wonder if `ObservableConvertibleType` will ever be a useful protocol for 3rd party users :sweat_smile: